lib/pry-remote.rb

```diff
@@ -260,26 +284,10 @@ def uri
   # Parses arguments and allows to start the client.
   class CLI
     def initialize(args = ARGV)
-      params = Slop.parse args, :help => true do
-        banner "#$PROGRAM_NAME [OPTIONS]"
-
-        on :s, :server=, "Host of the server (#{DefaultHost})", :argument => :optional,
-           :default => DefaultHost
-        on :p, :port=, "Port of the server (#{DefaultPort})", :argument => :optional,
-           :as => Integer, :default => DefaultPort
-        on :w, :wait, "Wait for the pry server to come up",
-           :default => false
-        on :r, :persist, "Persist the client to wait for the pry server to come up each time",
-           :default => false
-        on :c, :capture, "Captures $stdout and $stderr from the server (true)",
-           :default => true
-        on :f, "Disables loading of .pryrc and its plugins, requires, and command history "
-      end
-
-      exit if params.help?
+      params = Parser.parse(args)
 
-      @host = params[:server]
-      @port = params[:port]
+      @host = params[:server] || DefaultHost
+      @port = params[:port] || DefaultPort
```

Comment:
Parse class should define the default values. Let's improve this one in a follow-up PR.